packages/docs-reanimated/docs/fundamentals/getting-started.mdx

```diff
@@ -91,7 +91,32 @@ To learn more about the plugin head onto to [Reanimated babel plugin](/docs/fund
 
 </details>
 
-### Step 3: Clear Metro bundler cache (recommended)
+### Step 3: Wrap metro config with reanimated wrapper (recommended)
+
+Wrap your existing Metro configuration in the `metro.config.js` file with the `wrapWithReanimatedMetroConfig` function.
+
+```js
+const {
+  wrapWithReanimatedMetroConfig,
+} = require('react-native-reanimated/metro-config');
+
+const config = {
+  // Your existing Metro configuration options
+};
+
+module.exports = wrapWithReanimatedMetroConfig(config);
+```
+
+<details>
+<summary>Why should I do this?</summary>
+
+By wrapping your Metro configuration with the Reanimated Metro config wrapper, you can enhance reanimated errors and warnings with more accurate call stacks. This makes it easier to identify issues within your code.
```

Comment:
This sentence sound like the user helps Reanimated, because he _enhances reanimated error and warnings_. Let's reword it so we'd signify the benefit to the user more, i.e.:

```suggestion
Errors and warnings you encounter when you develop an app with Reanimated are sometimes indirect and difficult to understand. Reanimated Metro config wrapper handles these indirections internally and instead guides you directly to the offending code, providing clean and comprehensible information.

Unfortunately, we can't enable the wrapper in your app by default. 
```

Reply:
Too wordy for me. I can agree that the fragment _you can enhance_ of the sentence could be improved but I don't like your suggestion. I will think of something else.

Reply:
Good for me 👍